py2pack/__init__.py

```diff
@@ -311,29 +346,41 @@ def generate(args):
     print('generating spec file for {0}...'.format(args.name))
     data = args.fetched_data['info']
     durl = newest_download_url(args)
-    data['source_url'] = (args.source_url or
-                          (durl and durl['url']) or
-                          args.name + '-' + args.version + '.zip')
+    source_url = data['source_url'] = (args.source_url or (durl and durl['url']))
```

Comment:
This change is just a movement of code and adds the lines 382 and 383. I think it's not needed, just keep it as it is.

Reply:
Sometimes we place our generated source files under different file names, provided by --source-glob parameter. it would be nice to take this into account in our code.